src/Parsers/ASTSystemQuery.h

```diff
@@ -102,6 +102,7 @@ class ASTSystemQuery : public IAST, public ASTQueryWithOnCluster
         CANCEL_VIEW,
         TEST_VIEW,
         UNLOAD_PRIMARY_KEY,
+        LOAD_PRIMARY_KEY,
```

Comment:
```cpp
        LOAD_PRIMARY_KEY,
        UNLOAD_PRIMARY_KEY,
```

Reply:
Fixed

Reply:
I still see the `LOAD_PRIMARY_KEY` after the `UNLOAD_PRIMARY_KEY`